xds/src/main/java/io/grpc/xds/XdsServerWrapper.java

```diff
@@ -640,15 +642,16 @@ private boolean isPermanentError(Status error) {
       List<VirtualHost> virtualHosts = routingConfig.virtualHosts().get();
       if (virtualHosts == null) {
         String errorMsg = "Missing xDS routing config VirtualHosts due to RDS config unavailable.";
+        logger.log(Level.WARNING, errorMsg);
         call.close(Status.UNAVAILABLE.withDescription(errorMsg), new Metadata());
         return new Listener<ReqT>() {};
       }
       VirtualHost virtualHost = RoutingUtils.findVirtualHostForHostName(
           virtualHosts, call.getAuthority());
       if (virtualHost == null) {
-        call.close(
-                Status.UNAVAILABLE.withDescription("Could not find xDS virtual host matching RPC"),
-                new Metadata());
+        String errorMsg = "Could not find xDS virtual host matching RPC";
+        logger.log(Level.WARNING, errorMsg);
```

Comment:
What gRFC language caused you to add this? I don't think we should log for this case. Ditto below for no matching route.